hera_pspec/pspecdata.py

```diff
@@ -833,6 +835,20 @@ def set_Ndlys(self, ndlys=None):
                 raise ValueError("Cannot estimate more delays than there are frequency channels")
             self.spw_Ndlys = ndlys
 
+    def set_pol(self, pol=None):
```

Comment:
I tested it with cross-polarizations. If the beam is not provided, it seems to work (`get_Q` essentially takes a unit response beam matrix with a warning). If the beam is provided, it only works for co-polarizations and not cross. It shares the same error message as the existing normalization method does, if beam as well as cross-pol terms are provided.

Reply:
OK, that's good -- it may break in future, but it seems fine for now. What happens if the user wants to calculate two different sets of same-polarisation spectra from the same `PSpecData` object though? I think we currently support getting (e.g.) `(xx, xx)` and `(yy, yy)` from the same `PSpecData` object for example; does that break with these new changes?

Reply:
I suppose my main complaint is that I don't like the `PSpecData` object having a polarisation 'state' when it can handle input data with multiple polarisations. I think there is a way to do this without having to set the polarisation for the whole `PSpecData` object.

Reply:
Thanks. I see the issue now. As of now, I will simply try to pass the polarization to `get_Q` instead of setting the state.

Reply:
In the latest commit, polarization is passed on separately. It is used only when `exact_norm` is `True`. I have removed the `set_pol` function now.

Reply:
Thanks!